frontend/apps/reader/modules/readerhyphenation.lua

```diff
@@ -2,7 +2,7 @@ local InputContainer = require("ui/widget/container/inputcontainer")
 local UIManager = require("ui/uimanager")
 local JSON = require("json")
 local InfoMessage = require("ui/widget/infomessage")
-local ConfirmBox = require("ui/widget/confirmbox")
+local MultiConfirmBox = require("ui/widget/multiconfirmbox")
 local T = require("ffi/util").template
 local _ = require("gettext")
 local util = require("util")
```

Comment:
Would you mind reordering these? After having done many of them unordered requires are increasingly becoming an eyesore to me. :-P